README.md

```diff
@@ -231,6 +231,7 @@ The following options can be changed
 - `stopOnComplete` (type:boolean) - automatically call `stop()` when the value reaches the total (default: false)
 - `clearOnComplete` (type:boolean) - clear the progress bar on complete / `stop()` call (default: false)
 - `barsize` (type:int) - the length of the progress bar in chars (default: 40)
+- `autoExpand` (type:boolean) - ignores `barsize` and expands the bar to the window width (default: false)
```

Comment:
maybe `barsize`='auto' is more convenient ?

Reply:
Yes possible too, or set barsize to 0 or Infinity